src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.RyuJit.cs

```diff
@@ -2001,7 +2001,10 @@ private int SizeOfPInvokeTransitionFrame
                 int size = 5 * this.PointerSize;
 
                 if (_compilation.TypeSystemContext.Target.Architecture == TargetArchitecture.ARM)
+                {
                     size += this.PointerSize; // m_ChainPointer
```

Comment:
`m_PreserverRegs - RSP` comment above is not accurate for Arm now.

Delete unused `m_ChainPointer` while you are on it?

Reply:
Done. Had to rebase to get previous changes.